tests/validation/upgradecluster_test.go

```diff
@@ -0,0 +1,380 @@
+package validation
+import (
+	. "./testutils"
+	"flag"
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+
+	"strings"
+	"testing"
+	"time"
+)
+
+var destroy = flag.Bool("destroy", false, "a bool")
+var nodeOs = flag.String("node_os", "centos8", "a string")
+var externalDb = flag.String("external_db", "mysql", "a string")
+var arch = flag.String("arch", "amd64", "a string")
+var clusterType = flag.String("cluster_type", "etcd", "a string")
+var resourceName = flag.String("resource_name", "etcd", "a string")
+var sshuser = flag.String("sshuser", "ubuntu", "a string")
+var sshkey = flag.String("sshkey", "", "a string")
+var upgradeVersion = flag.String("upgradeVersion", "", "a string")
+
+
+var (
+	Kubeconfig string
```

Comment:
Are these variables used outside of the validation package? If not, please unexport them. (lowercase the first letter)